app/templates/application.hbs

```diff
@@ -1,3 +1,16 @@
-<main>
-  {{outlet}}
-</main>
+<header id="topbar">
+  <h1>Kosmos Kredits</h1>
+
+  <section id="user-account">
+    {{#if kredits.hasAccounts }}
```

Comment:
Can we make it
```hbs
{{#if (and kredits.hasAccounts kredits.currentUser}}
{{else}
```

Reply:
Sounds to me like that should be a single computed property.

Reply:
As far as I understand the code we already handle `hasAccounts` in `getCurrentUser`:
https://github.com/67P/kredits-web/blob/master/app/services/kredits.js#L147-L162